deploy/bundle.yaml

```diff
@@ -16,6 +16,9 @@ spec:
     - name: v1
       storage: true
       served: true
+    - name: v1-1-0
+      storage: false
+      served: true
```

Comment:
not sure whether we need this.
@delgod do we?

Reply:
technically not
but let's add, reason - to have all versions consistent

Reply:
the point is I create bundle automatically, so if I'll need to add exception on every release it's a bit painful, so I vote to make it consistent and easy to maintain